registry/internal/harbor/bootstrap.go

```
@@ -0,0 +1,412 @@
+	}
+	return c
+}
+
+// Ping checks whether Harbor is up and accepting requests.
+func (c *Client) Ping(ctx context.Context) error {
+	req, _ := http.NewRequestWithContext(ctx, "GET", c.baseURL+"/api/v2.0/ping", nil)
+	resp, err := c.http.Do(req)
+	if err != nil { //Pods are not started yet
+		return err
+	}
+	defer resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)